config_management/roles/kubernetes-start/tasks/main.yml

```diff
@@ -11,19 +11,19 @@
     enabled: yes
 
 - name: kubeadm init on the master
-  command: 'kubeadm init --token={{ kubernetes_token }}'
+  command: 'kubeadm init'
   when: ' {{ play_hosts[0] == inventory_hostname }}'
 
 - name: allow pods to be run on the master (if only node)
-  command: kubectl taint nodes --all dedicated-
+  command: kubectl --kubeconfig /etc/kubernetes/admin.conf taint nodes --all node-role.kubernetes.io/master:NoSchedule-
```

Comment:
Only the kubeconfig is gonna be backwards compatible. Master taint will only work with 1.6 and above.